config/environments/development.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :amazon
+  config.active_storage.service = :local
```

Comment:
I believe changing back to local for development is going to cause us pain now that we want to rely on the files existing in S3.  We moved away from ActiveStorage bitstreams because they were super slow.  Adding them back in seems like a mistake.